db_access.py

```diff
@@ -1,9 +1,23 @@
 import sqlite3
-
+import uuid, datetime
 from flask import app, g
 
 from chirpr import app
 
+def add_chirp(text):
+    conn = get_db()
+    num = -1
+    for c in conn.execute('SELECT id FROM chirp'):
+        num = c[0]
+    num += 1
+    conn.execute('''
+        INSERT INTO chirp VALUES (?,?,?,?)
+    ''', (num, text, 1, str(datetime.datetime.utcnow())))
+    conn.commit()
+    return num
+    #c.id, c.body, c.datetime, u.handle FROM chirp c, user u WHERE c.user_id = u.id
```

Comment:
This method is overly complicated, and inefficient. You wouldn't want to iterate over every `id` every time you add a chirp.

Look at `add_user`, and note that it doesn't specify the id -- which is totally OK, the DB handles it. This is the effect of `AUTOINCREMENT` in the `CREATE TABLE` statement.